docs/src/content/docs/id/guides/sidebar.mdx

```diff
@@ -265,54 +317,55 @@ Konfigurasi di atas akan menghasilkan sidebar berikut:
 <SidebarPreview
 	config={[
 		{
-			label: 'Panduan',
+			label: 'Stars',
 			items: [
 				{
-					label: 'Komponen',
-					link: '/id/guides/components/',
-					badge: { text: 'Baru', variant: 'default' },
+					label: 'Persei',
+					link: '',
+					badge: { text: 'Supergiant', variant: 'default' },
 				},
 			],
 		},
 		{
-			label: 'Referensi',
-			badge: { text: 'Ditinggalkan', variant: 'default' },
+			label: 'Moons',
+			badge: { text: 'Outdated', variant: 'default' },
 			items: [
 				{
-					label: 'Referensi Konfigurasi',
-					link: '/id/reference/configuration/',
+					label: 'Io',
+					link: '',
 				},
 				{
-					label: 'Referensi Frontmatter',
-					link: '/id/reference/frontmatter/',
+					label: 'Europa',
+					link: '',
 				},
 				{
-					label: 'Referensi Overrides',
-					link: '/id/reference/overrides/',
+					label: 'Ganymede',
+					link: '',
 				},
 			],
 		},
 	]}
 />
 
-### Variasi badge
+### Variasi badge dan tampilan kustom
 
-Sesuaikan gaya lambang menggunakan objek dengan properti `text` dan `variant`.
+Sesuaikan gaya badge menggunakan objek dengan properti `text`, `variant`, dan `class`.
 
 `text` mewakili konten yang akan ditampilkan (misalnya "Baru").
-Timpa gaya `default`, yang menggunakan warna aksen website Anda, dengan mengatur properti `variant` ke salah satu nilai berikut: `note`, `tip`, `danger`, `caution`, atau `success`.
+Secara default, badge akan menggunakan warna aksen website Anda. Untuk menggunakan tampilan badge bawaan, tetapkan properti `variant` ke salah satu nilai berikut: `note`, `tip`, `danger`, `caution` atau `success`.
 
-```js
+Secara opsional, Anda dapat membuat tampilan badge kustom dengan menyetel properti `class` ke nama kelas CSS.
+
+```js {9}
 starlight({
 	sidebar: [
 		{
-			label: 'Panduan',
+			label: 'Stars',
 			items: [
-				// Sebuah tautan dengan badge "Eksperimental" berwarna kuning.
+				// Tautan dengan badge "Stub" berwarna kuning.
 				{
-					label: 'Komponen',
-					link: '/guides/components/',
-					badge: { text: 'Eksperimental', variant: 'caution' },
+					slug: 'stars/sirius',
+					badge: { text: 'Stub', variant: 'caution' },
```

Comment:
The badge and label can both be translated here and in the preview below.

Reply:
I see, thank you. will update soon.